A3A/addons/core/functions/CREATE/fn_AIVEHinit.sqf

```diff
@@ -21,6 +21,11 @@ if (fullCrew [_veh, "", true] isEqualTo []) exitWith {
 	if (typeof _veh in A3A_utilityItemHM) then { _veh call A3A_fnc_initObject };
 };
 
+_veh setVehicleRadar 0; ///might break something
```

Comment:
Well... does it break anything?

Reply:
Seems as no, If it would be 1, ALL vehicles will always use radar, which would make it either too easy or hard. So I left it in automatic mode. AI will decide to use radar or not.